imap_data_access/file_validation.py

```diff
@@ -534,6 +534,10 @@ class SPICEFilePath(ImapFilePath):
     # Frame: (type: 'tf')
     spice_frame_pattern = r"(imap)_(?P<version>[\d]+)\.(?P<type>tf)"
 
+    # Covers:
+    # Science Frame: (type: 'tf')
+    science_frame_pattern = r"(imap_science)_(?P<version>[\d]+)\.(?P<type>tf)"
```

Comment:
You can do something similar to DPS regex.
```suggestion
    science_frame_pattern = r"(?P<type>imap_science)_(?P<version>[\d]+)\.tf"
```